IBPSA/Fluid/HeatPumps/ModularReversible/Data/TableData2D/EN14511/SingleSplitRXM20R.mo

```diff
@@ -0,0 +1,57 @@
+within IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D.EN14511;
+record SingleSplitRXM20R "Daikin_FTXM20R_RXM20R heating mode"
+  extends
+    IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D.GenericHeatPump(
+    use_TConOutForOpeEnv=true,
+    use_TEvaOutForOpeEnv=false,
+    dpEva_nominal=0,
```

Comment:
Enter `dpEva_nominal` and `dpCon_nominal` from the manufacturer data sheet.
If not available, set them to zero in the example; otherwise users will simulate models with these data assuming they capture the pressure drop.